packages/mavryk-protocol-demo-noops/mavryk-protocol-demo-noops.20.2~rc2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "info@mavryk.io"
+authors: ["Mavryk Dynamics"]
+homepage: "https://mavrykdynamics.com/"
+bug-reports: "https://gitlab.com/mavryk-network/mavryk-protocol/issues"
+dev-repo: "git+https://gitlab.com/mavryk-network/mavryk-protocol.git"
+license: "MIT"
+depends: [
+  "dune" { >= "3.11.1" }
+  "ocaml" { >= "4.14" }
+  "mavkit-proto-libs" { = version }
+  "mavkit-shell-libs" { = version }
+]
+build: [
+  ["rm" "-r" "vendors" "contrib"]
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+available: os != "win32"
+synopsis: "Mavryk protocol demo-noops package"
```

Comment:
Another point of concern with this set of packages is how many of these are of value to the community of package users. Our policy policies stipulates that [packages should provide utility](https://github.com/ocaml/opam-repository/wiki/Policies#3-packages-should-provide-some-sort-of-utility-for-users). I'm unsure that "noop" demo packages can be seen to provide utility to users of this repository. Moreover, since the synopses of many these packages are vague or esoteric (e.g., "Mavryk: `mavryk-dac-node` library") it is hard to determine what use they would be to users.